source/includes/options-mongo.yaml

```diff
@@ -55,40 +55,44 @@ name: host
 directive: option
 args: <hostname>
 description: |
-  Specifies the name of the host machine where the
-  {{instance}} is running. If this is not specified,
-  {{program}} attempts to connect to a MongoDB process running on
-  the localhost.
+  Specifies the name of the host machine where the {{instance}} is
+  running. If this is not specified, {{program}} attempts to connect to
+  a MongoDB process running on the localhost.
 
-  To connect to a replica set, specify the :setting:`replica set name
-  <~replication.replSetName>` and a seed list of set members. Use the
-  following form:
+  To connect to a host machine, specify one of the following:
 
-  .. code-block:: none
+  - The *hostname* of a specific machine
 
-     <replSetName>/<hostname1><:port>,<hostname2><:port>,<...>
+    .. code-block:: none
 
-  For TLS/SSL connections (``--ssl``), {{program}} verifies that the
-  hostname of the {{instance}} to which you are connecting matches
-  the CN or SAN of the {{instance}}'s ``--sslPEMKeyFile`` certificate.
-  If the hostname does not match the CN/SAN, {{program}} will fail to
-  connect.
+       <hostname>[:port]
 
-  For `DNS seedlist connections <https://docs.mongodb.com/manual/reference/connection-string/#dns-seedlist-connection-format/>`_, specify the connection protocol as
-  ``mongodb+srv``, followed by the DNS SRV hostname record and any
-  options. The  ``authSource`` and ``replicaSet`` options, if included in
-  the connection string, will override any corresponding DNS-configured options
-  set in the TXT record. Use of the ``mongodb+srv:`` connection string implicitly
-  enables TLS/SSL (normally set with ``ssl=true``) for the client connection. The
-  TLS/SSL option can be turned off by setting ``ssl=false`` in the query string.
+  - A *replica set name* and the *hostname* of at least one member of
+    the replica set.
 
-  Example:
+    .. code-block:: none
 
-  .. code-block:: none
+       <replSetName>/<hostname1>[:<port>][,<hostname2>[:<port>],...]
+
+  - A valid :ref:`mongodb connection string <mongodb-uri>`.
+
+    .. code-block:: sh
+
+       mongodb://[username:password@]<hostname1>[:port][,<hostname2>[:port]...][/[database][?options]]
 
-     mongodb+srv://server.example.com/?connectionTimeout=3000ms
+  - A :ref:`DNS seedlist connection string <connections-dns-seedlist>`.
```

Comment:
We should note this is 3.6+ only, just in case users think this has been around for longer. e.g. `.. versionadded: 3.6`

Reply:
Oops this was already in there and I deleted it. Added the versionadded